### ISSUE TITLE
Installing .config file to another location

### ISSUE THREAD

**Issue by BlagoCuljak** [Squirrel/Squirrel.Windows#1550]
Hi there, I have an WPF app that uses .config files in which are located URL to API services to connect.

If I update application, it runs over the .config, and URL is not valid for that user anymore.

If I move [selected .config file to another folder as suggested here,](https://github.com/Squirrel/Squirrel.Windows/issues/921) how can I put the config file to some other folder outside of installation of Squirrel folder?

Do I need to connect to users and put the config file in some folder, each and every time someone installs my app?

**Comment by Thieum** [comment]
You can set the app.config to another arbitrary file that could be located anywhere outside of the install folder: https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/appsettings/appsettings-element-for-configuration

**Comment by BlagoCuljak** [comment]
Yes, I understand that, but do I need to put that file out of install folder, manually?

**Comment by Thieum** [comment]
You can create it from your program if it doesn't exist (first use of your program) with default values. If it does exist (next use), you can load it and read the config. It can be done without going the AppSettings way, with a file that you manage yourself.
